### PR TITLE
Create `Button` component

### PR DESCRIPTION
**Pull request by daoanhhuy26012001**
**The Button component:**
- [x] Implement creating the `button` element
- [x] Add `if` condition to display `image` or `text` + `icon`
- [x] Set style for each button type and add `button:hover`
- See more details: [Link](https://docs.google.com/document/d/1iTrT9cdKrMCjUc6aDLS514gEk7BUFj-m2SCs2wCy2Lw/edit#heading=h.7t5pbcmau6ih)

### CODE REVIEW

File: ReactJS/practice-one/src/components/Common/Button/index.jsx [hunk 1]
```diff
@@ -16,14 +16,28 @@ export const Button = ({
...
+  isRight,
   onClick
 }) => {
   return (
     <button
       type="button"
       onClick={onClick}
       className={['button', `button-${size}`, `button-${variant}`].join(' ')}>
-      {url ? <img src={url} /> : text}
+      {url ? (
```
Comment (phattruong-agilityio) [inline ReactJS/practice-one/src/components/Common/Button/index.jsx:28]: It should be a `imageURL`

File: ReactJS/practice-one/src/components/Common/Button/index.jsx [hunk 2]
```diff
@@ -16,14 +16,28 @@ export const Button = ({
...
+        <img src={url} />
+      ) : isRight ? (
```
Comment (phattruong-agilityio) [inline ReactJS/practice-one/src/components/Common/Button/index.jsx:30]: this one can be done by using flex css [flex-direction](https://css-tricks.com/almanac/properties/f/flex-direction/)
btw, isRight is not clear naming

File: ReactJS/practice-one/src/components/Common/Button/index.jsx (hunk 1 above)
Comment (daoanhhuy26012001) [reply ReactJS/practice-one/src/components/Common/Button/index.jsx:28]: Done

File: ReactJS/practice-one/src/components/Common/Button/index.jsx (hunk 2 above)
Comment (daoanhhuy26012001) [reply ReactJS/practice-one/src/components/Common/Button/index.jsx]: Done

**Review by phattruong-agilityio — APPROVED** [review]

**Pull request merged**